examples/simple_project_no_secondary/main.tf

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
Probably we could compress this into the simple project example and move secondary ranges into a "complex" example.

Reply:
I will fold that into it and restructure some of the examples accordingly